escoot/de-at/augment-gemeinsamer-service-und-mietvertrag-de-at.md

```diff
@@ -178,7 +172,7 @@ Der **Service** und seine Merkmale oder die geltenden Vertragsbedingungen könne
 
 Basiert die Änderung auf einer Gesetzesänderung oder einer behördlichen Entscheidung, ist **Augment** berechtigt, die Änderung ab dem Datum des Inkrafttretens der Gesetzesänderung oder der behördlichen Entscheidung umzusetzen.
 
-<div class="page"></div>
+<div class="page"/>
```

Comment:
Page breaks should follow [special markup](https://github.com/augment-eco/terms-and-conditions?tab=readme-ov-file#special-markup).

```suggestion
<div class="page"></div>
```